portfolio/src/main/webapp/script.js

```diff
@@ -67,3 +72,19 @@ function getMax() {
 function deleteComment() {
   fetch("/delete-list", {method: 'POST'})
 }
+
+function getMax() {
+  const maxElement = document.getElementById("maxwant");
+  const max = maxElement.options[maxElement.selectedIndex].value;
+  return max;
+}
+
+function deleteComment() {
+  fetch("/delete-list", {method: 'POST'})
+}
```

Comment:
nit: Seems like this is a duplicate of the code in lines 72-74?

Reply:
addressed